examples/simple-platform/configure.adoc

```diff
@@ -0,0 +1,251 @@
+= Configure the Example
+
+This section describes how to configure the example.
+
+== Create a GitHub OAuth application
+
+The Accounts app supports account creation and authentication via GitHub
+only. In order to complete an OAuth flow with GitHub as the identity
+provider, you need to create a new GitHub OAuth app and configure it to
+connect to the `web-auth-endpoint` node.
+
+. Visit link:https://github.com/[github.com] and log into your account.
++
+image:img/screen-github-home.png["The GitHub home page", 1280, 311,
+role="frame"]
+
+. Navigate to Settings > Developer Settings > OAuth Apps.
++
+image:img/screen-github-oauth_apps.png["The OAuth apps screen on
+GitHub", 1280, 303, role="frame"]
+
+. image:img/button-github-register_new_app.png["The Register a new
+  application button on GitHub", 207, 34, role="right"]
+  Click the **Register a new application** button. The **Register a new
+  OAuth application** screen is displayed:
++
+image:img/screen-github-register_oauth.png["The Register a new OAuth
+application screen on GitHub", 1280, 623, role="frame"]
+
+. In the **Application name** field, enter a name of your choice. For
+  example, `Interbit Test App`.
+
+. In the **Homepage URL** field, enter a URL. Any URL works since we are
+  running the example locally.
+
+. In the **Authorization callback URL** field, enter
+  `http://localhost:8888/oauth/github`.
++
+NOTE: The callback URL is case sensitive, and the port specified is for
+running the `web-auth-endpoint` node locally.
+
+. image:img/button-github-register_app.png["The Register app button on
+  GitHub", 163, 34, role="right"]
+  Click the **Register application** button to finish creating the OAuth
+  app.
++
+IMPORTANT: Leave this browser tab open as we are going to copy values
+from this tab in subsequent steps.
+
+
+== Clone the `interbit` repository and install `interbit-cli`
```

Comment:
This is a minor point, but the font size for `interbit` and `interbit-cli` in these subheadings looks larger than the regular subheading text. It makes the headings feel off balance, visually.